### DISCUSSION TITLE
bump repo for new zstd 1.5.4 ?

### DISCUSSION THREAD

**Discussion by sedlund** [ScoopInstaller/Main#4457] [Q&A] [answered]
I thought versions were auto updates by bot, so I've been waiting for an update.  Should this version update occur automatically?

https://github.com/facebook/zstd/releases

**Comment by rashil2000** [comment, marked as answer]
Updated: https://github.com/ScoopInstaller/Main/commit/e9ee17d68d1223d1b6d17195217d4b164a0de09e

**Comment by sedlund** [comment]
Did the bot fail because there is no 32bit release?

**Reply by r15ch13** [reply]
Yep

**Reply by sedlund** [reply]
Cool thanks!
